lib/animas/animasSimulator.js

```diff
@@ -153,7 +153,7 @@ exports.make = function(config){
           // categorize the basal we're processing as a suspend basal if its rate
           // is zero and its start falls within the suspend event we have on deck
           var resumed = Date.parse(currStatus.time) + currStatus.duration;
-          if (resumed > Date.parse(event.time)) {
+          if (resumed >= Date.parse(event.time)) {
```

Comment:
Not completely relevant to this PR, but as it was a one-character change I didn't want to create a whole separate PR. Basically, since we only have one-minute resolution, the suspend/resume event and the basal can stop at exactly the same time (but they can also be up to three minutes apart).